prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -360,6 +362,39 @@ class Hl7Serializer(val metadata: Metadata) {
                     terser.set(pathSpec, "") // Not at all sure what to do here.
                 }
             }
+            Element.Type.EMAIL -> {
+                if (value.isNotEmpty()) {
+                    setEmailComponent(terser, value, pathSpec, element)
+                }
+            }
+            Element.Type.DATETIME -> {
+                // HACK WARNING
+                // some states have complained that the specimen_collected_datetime and the
```

Comment:
Can we do this hack with a mapper? It's place where tricky stuff should live. Not in serializers.